app/js/controllers/foyer.js

```diff
@@ -32,8 +32,10 @@ angular.module('ddsApp').controller('FoyerCtrl', function($scope, $state, $state
             situation.individus = _.filter(situation.individus, function(individu) {
                 return 'conjoint' !== individu.role;
             });
+        } // En cas de parent isolé, on pose une question supplémentaire
```

Comment:
This comment is linked to the line below. It should either be appended at the end of the line it describes, or on its own line right above it. Here, it looks like it explains the closing curly brace.